scripts/webpack-bundle.js

```diff
@@ -40,6 +42,128 @@ function bundler (opts) {
+          "aproba",
+          "archy",
+          "cacache",
+          "call-limit",
+          "bluebird",
+          "bin-links",
+          "byte-size",
+          "chownr",
+          "cli-table2",
+          "cmd-shim",
+          "columnify",
+          "config-chain",
+          "debuglog",
+          "detect-indent",
+          "detect-newline",
+          "dezalgo",
+          "editor",
+          "figgy-pudding",
+          "find-npm-prefix",
+          "fs-vacuum",
+          "fs-write-stream-atomic",
+          "gentle-fs",
+          "glob",
+          "graceful-fs",
+          "has-unicode",
+          "hosted-git-info",
+          "iferr",
+          "imurmurhash",
+          "inflight",
+          "inherits",
+          "ini",
+          "init-package-json",
+          "is-cidr",
+          "json-parse-better-errors",
+          "JSONStream",
+          "lazy-property",
+          "libcipm",
+          "libnpmhook",
+          "libnpx",
+          "lockfile",
+          "lodash._baseindexof",
+          "lodash._baseuniq",
+          "lodash._bindcallback",
+          "lodash._cacheindexof",
+          "lodash._createcache",
+          "lodash._getnative",
+          "lodash.clonedeep",
+          "lodash.restparam",
+          "lodash.union",
+          "lodash.uniq",
+          "lodash.without",
+          "lru-cache",
+          "meant",
+          "mkdirp",
+          "mississippi",
+          "move-concurrently",
+          "nopt",
+          "normalize-package-data",
+          "npm-audit-report",
+          "npm-cache-filename",
+          "npm-lifecycle",
+          "npm-install-checks",
+          "npm-package-arg",
+          "npm-packlist",
+          "npm-pick-manifest",
+          "npm-profile",
+          "npm-registry-client",
+          "npm-registry-fetch",
+          "npm-user-validate",
+          "npmlog",
+          "once",
+          "opener",
+          "osenv",
+          "pacote",
+          "path-is-inside",
+          "promise-inflight",
+          "query-string",
+          "qrcode-terminal",
+          "qw",
+          "read",
+          "read-cmd-shim",
+          "read-installed",
+          "read-package-json",
+          "read-package-tree",
+          "readable-stream",
+          "readdir-scoped-modules",
+          "request",
+          "retry",
+          "rimraf",
+          "semver",
+          "sha",
+          "slide",
+          "sorted-object",
+          "sorted-union-stream",
+          "ssri",
+          "strip-ansi",
+          "tar",
+          "text-table",
+          "uid-number",
+          "umask",
+          "unique-filename",
+          "unpipe",
+          "uuid",
+          "validate-npm-package-license",
+          "validate-npm-package-name",
+          "which",
+          "wrappy",
+          "write-file-atomic",
+          "safe-buffer",
+          "tiny-relative-date",
+          "cli-columns",
+          "node-gyp"
+        ]
+      })
+    ],
```

Comment:
Can you make this `require('../package.json').bundleDependencies.filter(...)` so we know which specific packages are being blacklisted?

Reply:
Also, does this track transitive dependencies?

Reply:
Oh yeah thats a great idea! I'll do so.